R/Sound_DMR.R

```diff
@@ -963,6 +1027,9 @@ plot_changepoints <- function(data, changepoint_obj, gene_name, penalty_val,
 #' @param z_col (str) - the column to run the changepoint analysis on. This
 #' can be any column, but for DMR analysis we recommend using the z scores for
 #' a fixed effect variable.
+#' @param whole_genome (boolean) - whether to run changepoints on a whole genome
```

Comment:
nit: Could also make this parameter `subset_by="Gene" or "Chromosome"` (or any other column for that matter )